SDK Integration: OpenAI Agents  + Client .dxt (#146)
kluster-ai/docs: get-started/mcp/client-integrations.md

## Prerequisites
      
Before integrating with any client, ensure you have the required credentials:
      
--8<-- 'text/kluster-api-onboarding.md'
- **MCP token** (for some clients): Follow the [platform guide](/get-started/mcp/cloud/platform/) to enable MCP and obtain your token

!!! info "Which credential do I need?"
    Depending on the client and configuration method, you may need either your **kluster.ai API key** or your **MCP token**. Each configuration section below specifies which credential to use.

## Configuration by client

=== "Claude desktop (.dxt file)"

    The easiest way to add kluster Verify to Claude desktop is using the .dxt extension file:

    1. **Download** the extension: 
       
        [Download kluster-verify-mcp.dxt](/get-started/mcp/resources/kluster-verify-mcp.dxt){ .md-button .md-button--primary }

    2. **Install** in Claude desktop: Drag and drop the .dxt file onto Claude desktop Extension tab, then click **Install**.

        ![](/images/get-started/mcp/integrations/integrations-3.webp){ style="width:80%;" }

>     3. **Add** your API key: When prompted, enter your kluster.ai API key (get one from [Get an API key guide](/get-started/get-api-key/)).

Reviewer: ```suggestion
    3. **Add your API key**: When prompted, enter your kluster.ai API key (follow the [Get an API key guide](/get-started/get-api-key/){target=\_blank} to obtain one).
```

Grammarly.

        ![](/images/get-started/mcp/integrations/integrations-4.webp){ style="width:80%;" }

>     4. **Enable** extension and start using: The kluster Verify tools will be available immediately in your conversations.

Reviewer: ```suggestion
    4. **Enable the extension and start using**: The kluster Verify tools will be available immediately in your conversations.
```

Grammar.

        ![](/images/get-started/mcp/integrations/integrations-5.webp){ style="width:80%;" }

=== "Claude desktop (JSON config)"

    If you prefer manual configuration, you can add kluster Verify by editing Claude desktop's configuration file:

>     1. **Locate the configuration file**:

Reviewer: ```suggestion
    1. Locate the configuration file:
```

        - **macOS**: `~/Library/Application Support/Claude/claude_desktop_config.json`
        - **Windows**: `%APPDATA%/Claude/claude_desktop_config.json`

>     2. **Add** the MCP server configuration:

Reviewer: ```suggestion
    2. Add the MCP server configuration:
```

    
        ```json
        --8<-- 'code/get-started/mcp/get-started/get-started-1.json'
        ```

>     3. **Replace** YOUR_MCP_TOKEN with your actual MCP token (obtained after [enabling MCP](/get-started/mcp/cloud/platform/)).

Reviewer: ```suggestion
    3. Replace `YOUR_MCP_TOKEN` with your actual MCP token (obtained after [enabling MCP](/get-started/mcp/cloud/platform/){target=\_blank}) on the kluster.ai platform.
```

You're referring to something in code, so it should be wrapped in back ticks.

>     4. **Save** the file and **restart** Claude desktop to load the kluster Verify tools.

Reviewer: ```suggestion
    4. Save the file and restart Claude desktop to load the kluster Verify tools.
```

<!-- Commenting this for safekeeping -->
<!--See [Tools reference](/get-started/mcp/tools/){target=\_blank} for parameters and examples.-->

## SDK integrations

> Looking to integrate MCP tools into your own applications? Check out our SDK integration guides:

Reviewer: ```suggestion
Looking to integrate MCP tools into your own applications? Check out the SDK integration guides:
```

Reviewer: Avoid "we", "our", etc. in guides. Ok in tutorials, but should avoid everywhere else
